Ginger/GingerCoreNETUnitTest/RunTestslib/CLITest.cs

```diff
@@ -425,6 +425,46 @@ public void CLIDynamicJSON_ExistingCustomized_OnlyIDs_Test()
             Assert.AreEqual(((RunSetActionHTMLReportSendEmail)(WorkSpace.Instance.RunsetExecutor.RunSetConfig.RunSetActions[0])).Subject, "Test44", "Validating customized report mail Subject");
         }
 
+        [TestMethod]
+        // Testing if virtual environment is successfully used
+        public void CLIDynamicJSON_TestingVirtualEnvironment()
+        {
+
+            string jsonConfigFilePath = CreateTempJSONConfigFile(Path.Combine(TestResources.GetTestResourcesFolder("CLI"), "CLI-VirtualEnvironment.json"), mSolutionFolder);
+
+            CLIProcessor CLI = new ();
+
+            Task.Run(async () =>
+            {
+                await CLI.ExecuteArgs(new string[] { "dynamic", "-f", jsonConfigFilePath });
+            }).Wait();
+
+            Assert.AreEqual(WorkSpace.Instance.RunsetExecutor.RunsetExecutionEnvironment.Name , "Default123");
```

Comment:
Change environment name to Virutal environment and add assert for all properties